recipes/openvino/meta.yaml

```diff
@@ -0,0 +1,521 @@
+    {% for openvino_lib in openvino_libs %}  # [unix]
+    - test -f $PREFIX/lib/libopenvino{{ openvino_lib }}${SHLIB_EXT}  # [unix]
+    {% endfor %}  # [unix]
+
+    # C++ samples on Unix
+    - mkdir -p openvino_sources/samples/cpp/thirdparty  # [unix]
+    - cp -R openvino_sources/thirdparty/cnpy openvino_sources/samples/cpp/thirdparty/cnpy  # [unix]
+    - cmake -S openvino_sources/samples/cpp -B samples_cpp_build -DBIN_FOLDER=bin  # [unix]
+    - cmake --build samples_cpp_build --config Release --verbose  # [unix]
+    - ./samples_cpp_build/bin/hello_query_device  # [unix]
+
+    # C samples on Unix
+    - cp openvino_sources/samples/cpp/CMakeLists.txt openvino_sources/samples/c/CMakeLists.txt  # [unix]
+    - cmake -S openvino_sources/samples/c -B samples_c_build -DBIN_FOLDER=bin  # [unix]
+    - cmake --build samples_c_build --config Release --verbose  # [unix]
+
+    # C++ samples on Windows
+    - mkdir -p openvino_sources\\samples\\cpp\\thirdparty  # [win]
+    - cp -R openvino_sources\\thirdparty\\cnpy openvino_sources\\samples\\cpp\\thirdparty\\cnpy  # [win]
+    - cmake -S openvino_sources\\samples\\cpp -B samples_cpp_build -DBIN_FOLDER=bin  # [win]
+    - cmake --build samples_cpp_build --config Release --verbose  # [win]
+    - .\\samples_cpp_build\\bin\\Release\\hello_query_device.exe  # [win]
+
+    # C samples on Windows
+    - cp openvino_sources\\samples\\cpp\\CMakeLists.txt openvino_sources\\samples\\c\\CMakeLists.txt  # [win]
+    - cmake -S openvino_sources\\samples\\c -B samples_c_build -DBIN_FOLDER=bin  # [win]
+    - cmake --build samples_c_build --config Release --verbose  # [win]
+
+    # check that devices are found using OpenVINO C API
+    # TODO: fix for OSX and Linux64: currently, the application cannot be run because of RPATH
+    # - $CC snippets/openvino_available_devices.c $(pkg-config --cflags --libs openvino) -o openvino_devices_test  # [linux]
+    # - if [[ $(./openvino_devices_test) != "passed" ]]; then exit 1 ; fi  # [linux]
+
+    # check that frontends are found using OpenVINO C++ API
+    - cmake -S snippets -B snippets_build
+    - cmake --build snippets_build --config Release --verbose
+    - if [[ $(./snippets_build/openvino_frontends_test) != 4 ]]; then exit 1 ; fi  # [unix]
+
+
+outputs:
+  - name: libopenvino
+    version: {{ version }}
+    script: install/install.sh
+    about:
+      summary: OpenVINO Core Runtime library
+      license_file:
+        - LICENSE
+        - third-party-programs.txt
+        - onednn_third-party-programs.txt
+        - runtime-third-party-programs.txt
+        - tbb_third-party-programs.txt
+    build:
+      run_exports:
+        - {{ pin_subpackage('libopenvino', max_pin='x.x.x') }}
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ compiler('c') }}
+        - cmake >=3.13
+      host:
+        - libhwloc >=2.5 # starting from this version hwloc supports hybrid cores
```

Comment:
There is a static linking warning from libhwloc. Why?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=678394&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=5183